[2025-02-20T00:52:56.229+0000] {processor.py:186} INFO - Started process (PID=3655) to work on /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-20T00:52:56.230+0000] {processor.py:914} INFO - Processing file /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-20T00:52:56.232+0000] {logging_mixin.py:190} INFO - [2025-02-20T00:52:56.232+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-20T00:52:56.237+0000] {logging_mixin.py:190} INFO - [2025-02-20T00:52:56.236+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-20T00:52:56.238+0000] {logging_mixin.py:190} INFO - [2025-02-20T00:52:56.238+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-20T00:52:56.238+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-20T00:52:56.252+0000] {processor.py:208} INFO - Processing /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2025-02-20T00:54:01.640+0000] {processor.py:186} INFO - Started process (PID=3897) to work on /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-20T00:54:01.642+0000] {processor.py:914} INFO - Processing file /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-20T00:54:01.644+0000] {logging_mixin.py:190} INFO - [2025-02-20T00:54:01.644+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-20T00:54:01.648+0000] {logging_mixin.py:190} INFO - [2025-02-20T00:54:01.647+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-20T00:54:01.648+0000] {logging_mixin.py:190} INFO - [2025-02-20T00:54:01.648+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-20T00:54:01.648+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-20T00:54:01.661+0000] {processor.py:208} INFO - Processing /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2025-02-20T00:54:58.073+0000] {processor.py:186} INFO - Started process (PID=4139) to work on /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-20T00:54:58.074+0000] {processor.py:914} INFO - Processing file /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-20T00:54:58.076+0000] {logging_mixin.py:190} INFO - [2025-02-20T00:54:58.076+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-20T00:54:58.079+0000] {logging_mixin.py:190} INFO - [2025-02-20T00:54:58.079+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-20T00:54:58.080+0000] {logging_mixin.py:190} INFO - [2025-02-20T00:54:58.079+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-20T00:54:58.080+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-20T00:54:58.091+0000] {processor.py:208} INFO - Processing /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2025-02-20T00:55:53.434+0000] {processor.py:186} INFO - Started process (PID=4259) to work on /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-20T00:55:53.435+0000] {processor.py:914} INFO - Processing file /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-20T00:55:53.437+0000] {logging_mixin.py:190} INFO - [2025-02-20T00:55:53.437+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-20T00:55:53.441+0000] {logging_mixin.py:190} INFO - [2025-02-20T00:55:53.441+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-20T00:55:53.442+0000] {logging_mixin.py:190} INFO - [2025-02-20T00:55:53.442+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-20T00:55:53.442+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-20T00:55:53.455+0000] {processor.py:208} INFO - Processing /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2025-02-20T00:56:46.934+0000] {processor.py:186} INFO - Started process (PID=4380) to work on /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-20T00:56:46.936+0000] {processor.py:914} INFO - Processing file /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-20T00:56:46.938+0000] {logging_mixin.py:190} INFO - [2025-02-20T00:56:46.937+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-20T00:56:46.941+0000] {logging_mixin.py:190} INFO - [2025-02-20T00:56:46.941+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-20T00:56:46.942+0000] {logging_mixin.py:190} INFO - [2025-02-20T00:56:46.942+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-20T00:56:46.942+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-20T00:56:46.953+0000] {processor.py:208} INFO - Processing /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2025-02-20T00:57:41.201+0000] {processor.py:186} INFO - Started process (PID=4553) to work on /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-20T00:57:41.202+0000] {processor.py:914} INFO - Processing file /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-20T00:57:41.204+0000] {logging_mixin.py:190} INFO - [2025-02-20T00:57:41.204+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-20T00:57:41.208+0000] {logging_mixin.py:190} INFO - [2025-02-20T00:57:41.207+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-20T00:57:41.208+0000] {logging_mixin.py:190} INFO - [2025-02-20T00:57:41.208+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-20T00:57:41.208+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-20T00:57:41.220+0000] {processor.py:208} INFO - Processing /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2025-02-20T00:58:34.751+0000] {processor.py:186} INFO - Started process (PID=4668) to work on /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-20T00:58:34.753+0000] {processor.py:914} INFO - Processing file /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-20T00:58:34.755+0000] {logging_mixin.py:190} INFO - [2025-02-20T00:58:34.754+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-20T00:58:34.758+0000] {logging_mixin.py:190} INFO - [2025-02-20T00:58:34.758+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-20T00:58:34.758+0000] {logging_mixin.py:190} INFO - [2025-02-20T00:58:34.758+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-20T00:58:34.759+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-20T00:58:34.770+0000] {processor.py:208} INFO - Processing /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2025-02-20T00:59:27.104+0000] {processor.py:186} INFO - Started process (PID=4783) to work on /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-20T00:59:27.140+0000] {processor.py:914} INFO - Processing file /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-20T00:59:27.142+0000] {logging_mixin.py:190} INFO - [2025-02-20T00:59:27.142+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-20T00:59:27.146+0000] {logging_mixin.py:190} INFO - [2025-02-20T00:59:27.145+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-20T00:59:27.146+0000] {logging_mixin.py:190} INFO - [2025-02-20T00:59:27.146+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-20T00:59:27.146+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-20T00:59:27.159+0000] {processor.py:208} INFO - Processing /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.062 seconds
[2025-02-20T01:00:18.798+0000] {processor.py:186} INFO - Started process (PID=5246) to work on /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-20T01:00:18.800+0000] {processor.py:914} INFO - Processing file /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-20T01:00:18.802+0000] {logging_mixin.py:190} INFO - [2025-02-20T01:00:18.802+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-20T01:00:18.806+0000] {logging_mixin.py:190} INFO - [2025-02-20T01:00:18.805+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-20T01:00:18.806+0000] {logging_mixin.py:190} INFO - [2025-02-20T01:00:18.806+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-20T01:00:18.806+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-20T01:00:18.820+0000] {processor.py:208} INFO - Processing /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2025-02-20T01:01:16.476+0000] {processor.py:186} INFO - Started process (PID=5470) to work on /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-20T01:01:16.478+0000] {processor.py:914} INFO - Processing file /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-20T01:01:16.480+0000] {logging_mixin.py:190} INFO - [2025-02-20T01:01:16.480+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-20T01:01:16.483+0000] {logging_mixin.py:190} INFO - [2025-02-20T01:01:16.483+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-20T01:01:16.484+0000] {logging_mixin.py:190} INFO - [2025-02-20T01:01:16.484+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-20T01:01:16.484+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-20T01:01:16.498+0000] {processor.py:208} INFO - Processing /home/ubuntu/redfin_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
